chunky/src/java/se/llbit/chunky/chunk/BlockPalette.java

```diff
@@ -67,6 +65,9 @@ public BlockPalette(Map<BlockSpec, Integer> initialMap, List<Block> initialList)
     this.blockMap = initialMap;
     this.palette = initialList;
     this.materialProperties = getDefaultMaterialProperties();
+
+    putLegacyTags(); //must be done first as legacy blocks are index-dependant
```

Comment:
The `palette` defaults to `initialList`, which might not be empty. In this case, this can't be guaranteed.